<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(11, 12, 67, 150);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="imageLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>200</width>
     <height>180</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: none;
border: none;</string>
   </property>
   <property name="text">
    <string>image</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>370</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: none;
border: none;</string>
   </property>
   <property name="text">
    <string>Title</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="authorLabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>110</y>
     <width>280</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: none;
border: none;</string>
   </property>
   <property name="text">
    <string>Author</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="sourceLabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>150</y>
     <width>280</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: none;
border: none;</string>
   </property>
   <property name="text">
    <string>Source Site</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="authorLabel_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: none;
border: none;</string>
   </property>
   <property name="text">
    <string>By:</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="authorLabel_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: none;
border: none;</string>
   </property>
   <property name="text">
    <string>At</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
